stdpopsim/cli.py

```diff
@@ -530,9 +624,43 @@ def run_simulation(args):
             f"Running simulation model {model.id} for {species.id} on "
             f"{contig} with {len(samples)} samples using {engine.id}."
         )
+        if args.dfe is None:
+            dfe = None
+        # Including DFE (selection) in the simulation
+        else:
+            dfe = species.get_dfe(args.dfe)
+            # Including selection in a given interval
+            if args.dfe_interval is None:
+                # if not specified, whole contig is under selection
+                contig.add_dfe(
+                    intervals=np.array(
+                        [[0, int(contig.recombination_map.sequence_length)]]
+                    ),
+                    DFE=dfe,
+                )
+            # Including selection in a specific interval
+            else:
+                inter = args.dfe_interval.split(",")
+                contig.add_dfe(
+                    intervals=np.array([[int(inter[0]), int(inter[1])]]),
+                    DFE=dfe,
+                )
```

Comment:
Slightly more concise:
```suggestion
            if args.dfe_interval is None:
                left, right = 0, int(contig.recombination_map.sequence_length)
            else:
                left, right = args.dfe_interval.split(",")
            contig.add_dfe(
                intervals=np.array([[int(inter[0]), int(inter[1])]]),
                DFE=dfe,
            )
```